### ISSUE TITLE
Blessing of Protection (Elysium) (Zg Patch)

### ISSUE THREAD

**Issue by charlymi6** [LightsHope/server#228]
Hi, i would like report the same issue that happened after Bwl patch, and now come back again after zg patch. When you use Blessing of Protection on someone in vanilla (and a long time ago after on official server) you stop him to pick up a plant or a mine. Ok this is unfair, but this is vanilla rules. Last time i report that was fix quickly (in august). So now after Zul Gurb update, the same issue come back, Blessing of Protection don't stop to pick up a plant or a mine. Please fix it, it's Vanilla Blizzlike homemade ;)

**Comment by Fiskehatt** [comment]
http://wow.allakhazam.com/forum.html?forum=243&mid=1158254623130746425 (Sep 14 2006)

> Claimjump - cast Blessing of Protection on someone gathering, and then take their spot (note: incredibly rude)

> BoP only works on party/raid members. But it's tremendously fun to use on people who are trying to mine dark iron in MC. 

Seems that you can interrupt someone gathering with BoP. However, they naturally need to be in the same party/raid as stated by the tooltip so this has little significance in open world farming:

> A targeted **party member** is protected from all physical attacks for 6 sec, but during that time they cannot attack or use physical abilities. 

Anyways, the mechanic should probably be the same as for battleground flags (see issue #400). 

http://www.ownedcore.com/forums/world-of-warcraft/world-of-warcraft-exploits/633-arathi-basin-flag-capping-exploit.html:

> When your start capturing the flag in AB, simply have someone cast Blessing of Protection on you and right before you finish capturing it, simply cancel it out. This will prevent the BOP from cancelling out your flag capture.

In other words, the gathering/capturing should result in a failure, most likely at the end of the cast.
